docs/iris/example_code/General/polynomial_fit.py

```diff
@@ -0,0 +1,51 @@
+"""
+Fitting a polynomial
+====================
+
+This example demonstrates adding a polynomial fit to a 1D plot of data from
```

Comment:
I'd reword this because it sounds like the plot has to do with the fitting process (i.e. fitting to a plot). In fact the important part of this example is really the handling of metadata, so perhaps you could afford to be a little more verbose. Perhaps something along the lines of:

```
This example demonstrates computing a polynomial fit to 1D data from an Iris cube,
adding the fit to the cube's metadata, and plotting both the original data and the fit.
```